test/utils/data.test.js

```diff
@@ -603,10 +615,10 @@ describe('DataUtil', () => {
     });
 
     it('should return the avg daily total basal and bolus insulin delivery when viewing more than 1 day', () => {
-      dataUtil.endpoints = twoDayEndpoints;
+      dataUtil.endpoints = twoWeekEndpoints;
       expect(dataUtil.getBasalBolusData()).to.eql({
-        basal: 0.75,
-        bolus: 7.5,
+        basal: 1, // (0.25 + 0.75 + 0.5 + 0.5) / 2 days
+        bolus: 9.5, // (4 + 5 + 6 + 4) / 2
```

Comment:
This one is quite difficult to unterstand because it covers several cases: 
- only 2 days included in the 2 weeks
- several days in the calculation